backend/server/routes/getTechnicalProgression.js

```diff
@@ -0,0 +1,26 @@
+export default async function getTechnicalProgression(givenProductCode){
+    if (typeof givenProductCode !== "string" || !givenProductCode)
+        throw new Error("Product code value not acceptable")
+const url = `${process.env.BASE_URL}/TechnicalEducationProgression/${givenProductCode}`
+
+// note assumes similarity between get routes structure and ge progression
+//options might need to be different
```

Comment:
options do not in fact need to be different - remind me to show you the tests in bruno so you can follow that setup in future issues

and please get rid of these comments